src/concrete/ml/quantization/quantizers.py

```diff
@@ -771,7 +772,7 @@ def quant(self, values: numpy.ndarray) -> numpy.ndarray:
 
         return qvalues.astype(numpy.int64)
 
-    def dequant(self, qvalues: numpy.ndarray) -> Union[Any, numpy.ndarray]:
+    def dequant(self, qvalues: numpy.ndarray) -> numpy.ndarray:
```

Comment:
`-> Union[numpy.ndarray, Tracer]` maybe no ? surprised mypy did not complain

Reply:
Hum actually no, we never have `Tracer`s in the `dequant` function.

Reply:
but at the end you have : 
```
      assert isinstance(values, (numpy.ndarray, Tracer))
        return values
```
🤔

Reply:
Ho yeah indeed weird, I'll remove `Tracer` from the assert.

Reply:
Ha no okay we are in a quantizer my bad, I thought it was in `quantized_module.py` yes I'll fix the type hint.